lib/spack/spack/cmd/export.py

```diff
@@ -126,6 +126,7 @@ def export(parser, args):
             sflags = ' '.join(str(f) for f in sorted(sflags))
             bflags = ''.join(str(f) for f in sorted(bflags))
             key = ' '.join([e for e in (key, sflags, bflags) if len(e) > 0])
+            key = str(key)
```

Comment:
nitpick: Id do str() on the previous line' expression

Reply:
I wanted to keep the logic from being too nested…